src/azure-cli/azure/cli/command_modules/synapse/_help.py

```diff
@@ -919,11 +950,11 @@
   - name: Delete role assignments by role and assignee.
     text: |-
         az synapse role assignment delete --workspace-name testsynapseworkspace \\
-        --role "Sql Admin" --assignee sp_name
+        --role "Synapse Administrator" --assignee sp_name
   - name: Delete role assignments by role id/name.
     text: |-
         az synapse role assignment delete --workspace-name testsynapseworkspace \\
-        --role "Sql Admin"
+        --role "Synapse Administrator"
```

Comment:
Why no new examples for deleting role assignments by scope?

Reply:
Will add it.